model/src/test/assets/expected-scan-results.yml

```diff
@@ -15,9 +15,11 @@ results:
     startTime: "1970-01-02T00:01:00Z"
     endTime: "1970-01-02T00:02:00Z"
     fileCount: 1
-    licenses:
-    - "license 1.1"
-    - "license 1.2"
+    findings:
+      license 1.1:
```

Comment:
Just me 2 cents: in doing so, you no longer have self-describing named fields for the license and copyrights as you had before.
IMHO you should structure your data rather something like this way:
```
findings:
  -  license: license 2.1
     copyrights:
           - copyright 3
           - copyright 4
  -  license: license XYZ
     copyrights:
           - copyright 2
           - copyright 1

```

Reply:
Yes, good point, I should change that. Thanks!

Reply:
Done in the latest iteration of the PR.